src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((base1 + base2) / 2) * height;
```

Comment:
It is recommended to avoid using magic numbers directly in the code. In this case, the number '2' used in the area calculation is a magic number. Consider defining it as a constant with a meaningful name.